libgnucash/engine/gnc-pricedb.c

```diff
@@ -698,60 +698,38 @@ compare_prices_by_date(gconstpointer a, gconstpointer b)
                          gnc_price_get_guid((GNCPrice *) b));
 }
 
-typedef struct
+static gint
+find_dupl_price (const GNCPrice *a, const GNCPrice *b)
```

Comment:
`price_compare` or maybe `prcmp` if you want to pun `strcmp`. `find` implies that it's traversing a price list looking for a duplicate.